docs/en/siem/siem-ui.asciidoc

```diff
@@ -8,7 +8,31 @@ environment, and you can use the interactive UI to drill down into areas of
 interest.
 
 The *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* bar is available
-throughout the {siem-app} for searching and filtering.
+throughout the {siem-app} for searching and filtering. You can also click and
+swipe histograms, which updates the global time filter.
+
+[role="screenshot"]
+image::images/siem-click-swipe.png[]
+
+TIP: All {siem-soln} histograms and graphs contain an **Inspect** button, so
+you can examine data sources throughout the app.
```

Comment:
> so you can examine data sources throughout the app.

Consider replacing "data sources" with "queries", e.g.:

so you can view the Elasticsearch queries that provide data for graphs throughout the app.